docs/getting_started/Datus_BI.zh.md

```
@@ -0,0 +1,193 @@
+      sudo apt-get install helm
+      ```
+      2. fedora: `sudo dnf install helm`
+      3. Snap: `sudo snap install helm --classic`
+      4. FreeBSD: `pkg install helm`
+   3. windows
+      1. 使用Chocolatey `choco install kubernetes-helm` 
+      2. 使用Scoop `scoop install helm`
+      3. 使用Winget `winget install Helm.Helm`
+   > Linux和Mac系统也可以使用脚本安装： 
+    ```bash
+    $ curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-4
```

markdownlint-cli2 (0.18.1)

Dollar signs used before commands without showing output

(MD014, commands-show-output)